src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishANetCoreAppForTelemetry.cs

```diff
@@ -53,15 +53,17 @@ public void It_collects_Trimmer_SingleFile_ReadyToRun_publishing_properties(stri
             s.Should().Contain(
                 "{\"EventName\":\"PublishProperties\",\"Properties\":{\"PublishReadyToRun\":\"True\",\"PublishTrimmed\":\"True\",\"PublishSingleFile\":\"True\"}");
             s.Should().Contain(
-                "{\"EventName\":\"ReadyToRun\",\"Properties\":{\"PublishReadyToRunUseCrossgen2\":\"null\",\"Crossgen2PackVersion\":\"null\"");
+                "{\"EventName\":\"ReadyToRun\",\"Properties\":{\"PublishReadyToRunUseCrossgen2\":\"true\",")
+                .And.MatchRegex(
+                    "\"Crossgen2PackVersion\":\"[5-9]\\..+\"");
```

Comment:
@LakshanF @vitek-karas @agocke Could someone from CrossGen/ReadyToRun look at this and see if this test should be updated to target the "latest" target framework instead of just .NET 5?  Note that the assertion had to change slightly.

Reply:
@trylek as someone who actually knows things about R2R/CrossGen.